mne/io/curry/curryreader.py

```diff
@@ -0,0 +1,633 @@
+# Authors: The MNE-Python contributors.
+# License: BSD-3-Clause
+# Copyright the MNE-Python contributors.
```

Comment:
is this file the official reader? is this file copied from somewhere?

Reply:
yes, it's copied from https://github.com/neuroscan/curry-python-reader

the false info you flagged was added by [autofix.ci]

Reply:
you discussed this topic with a compumedics dev in #12855 

they said they wont supply a pypi or conda version, but we are free to use the code.
the github has a BSD3 license applied, but they dont include any note in the file itself